core/src/main/res/xml/provider_paths.xml

```diff
@@ -6,4 +6,7 @@
   <root-path
     name="external_files"
     path="/storage/" />
+  <files-path
+    name="debug"
+    path="logs/" />
```

Comment:
The file path was updated here but still the same problem persists, a several tries have been made for resolution but none of them helped.

Reply:
Have you tried this `<files-path name="files" path="/" />` ? or perhaps `/logs` or `/logs/`

Reply:
I've tried all the permutations of `logs` and `/` but none helped and right now used only `/` for the attribute `path` it too didn't help. I've read the docs multiple times but still unable to figure out where i'm falling short of things

Reply:
Have you been logging the uri you get returned and can you post it here?

you may also want to change
```
android:authorities="${applicationId}.fileprovider"
```

I am not sure if `applicationId` is getting resolved correctly or at all `${applicationId}` this type of syntax means that a manifestPlaceholder has been set and I don't see that being set at all from a quick search

Reply:
![inFileLogger](https://user-images.githubusercontent.com/56556919/86332112-abe84200-bc67-11ea-9110-80bc928007b4.png)

Reply:
"couldn't find meta-data for provider with authority .." means that the authority and the packageName you address it with don't match, the provider is defined in the core module so it could have the id `org.kiwix.kiwixmobile.core`, try hardcoding `org.kiwix.kiwixmobile.fileprovider` in the authority